src/libs/Network/SequentialQueue.ts

```diff
@@ -197,9 +204,30 @@ function isPaused(): boolean {
 // Flush the queue when the connection resumes
 NetworkStore.onReconnection(flush);
 
-function push(request: OnyxRequest) {
-    // Add request to Persisted Requests so that it can be retried if it fails
-    PersistedRequests.save(request);
+function push(newRequest: OnyxRequest) {
+    // If a request is already being processed, ignore it when looking for potentially conflicting requests
+    const requests = PersistedRequests.getAll();
```

Comment:
NAB but you can move this inside the `if (checkAndFixConflictingRequest)` so we don't create this variable unnecessarily if it won't be used